### PR TITLE
fix a typo in docs/reference/dockerd.md

### PR DESCRIPTION
**Pull request by luojie233**
Signed-off-by: Jie Luo <luo612@zju.edu.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
LGTM, but please consider squashing with your other similar PRs: https://github.com/docker/docker/pull/29570

**Comment by luojie233** [comment]
OK, I merged #29570

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: docs/reference/builder.md [hunk 1]
```diff
@@ -1754,6 +1754,6 @@ FROM ubuntu
 RUN echo moo > oink
 # Will output something like ===> 695d7793cbe4
 
-# You᾿ll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
+# You'll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
```
Comment (thaJeztah) [inline docs/reference/builder.md]: Looks like this is also in this PR https://github.com/docker/docker/pull/29583/files#diff-4fc21624aa5dadd153e2780910da7c7dR1757

**Pull request closed without merging**